### PR TITLE
QS:SAP Technical Review Feedback

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

This PR collects all the feedback from the QS:SAP technical review period (closes 9 October).

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Partial https://github.com/SUSE/spacewalk/issues/10772
Previously https://github.com/uyuni-project/uyuni-docs/pull/559

### CODE REVIEW

File: modules/quickstart-sap/pages/prepare-clients.adoc [hunk 1]
```diff
@@ -47,19 +47,25 @@ Do not use a client machine to host the shared storage disk.
 
 == Download the SAP Installation Software
 
-Download the {sles}{nbsp}15 with SAP installation media from {scc}, and save a copy on each clients.
-Ensure you have saved it in the same file path on each client.
+Download the SAP installation media and save a copy on each clients.
```
Comment (keichwa) [inline modules/quickstart-sap/pages/prepare-clients.adoc:50]:
```suggestion
Download the SAP installation media and save a copy on each client.
```

File: modules/quickstart-sap/pages/prepare-clients.adoc [hunk 2]
```diff
@@ -47,19 +47,25 @@ Do not use a client machine to host the shared storage disk.
...
+The software that you require will differ, depending on your environment.
+For example, if you are using HANA, you need the SAP HANA platform.
+If you are using Netweaver, you need several different packages.
```
Comment (keichwa) [inline modules/quickstart-sap/pages/prepare-clients.adoc:53]:
```suggestion
If you are using Netweaver, you need different packages.
```

**Review by keichwa — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]

**Comment by Loquacity** [comment]
image
image
image

**Comment by Loquacity** [comment]
Updated PDF: 
[Uploading suse_manager_quickstart-sap_guide.pdf…]()

**Comment by Loquacity** [comment]
Technical Review period closed.

**Pull request merged**